spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapter.java

```diff
@@ -137,4 +143,24 @@ public void setHeaderMapper(HeaderMapper<Map<String, String>> headerMapper) {
 		Assert.notNull(headerMapper, "The header mapper can't be null.");
 		this.headerMapper = headerMapper;
 	}
+
+	/**
+	 * Returns the {@link PubSubMessageConverter} used to convert the {@code byte[]} Pub/Sub
+	 * payload to an object of the configured payload type.
+	 * @since 1.1
+	 */
+	public PubSubMessageConverter getMessageConverter() {
+		return this.pubSubMessageConverter;
+	}
+
+	/**
+	 * Sets the {@link PubSubMessageConverter} used to convert the {@code byte[]} Pub/Sub
+	 * payload to an object of the configured payload type.
+	 * @since 1.1
+	 */
+	public void setMessageConverter(
```

Comment:
Please, move all the setters and getter just after ctors: `protected` and `private` method should go after: https://github.com/spring-projects/spring-framework/wiki/Code-Style#java-source-file-organization

The same requirement applies for the `PubSubMessageHandler` as well